### PR TITLE
My first PR to contributor-playground

### PR DESCRIPTION
**Pull request by wikix**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: shanghai/2020/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+  - wikix
```
Comment (jeefy) [inline shanghai/2020/OWNERS:4]: Apologies, but you cannot simply PR yourself into becoming an approver. You must first follow the contributor ladder by becoming an org member, then become a reviewer.

File: shanghai/2020/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# wikix
+Hello, I'm wikix from China
```
Comment (idealhack) [inline shanghai/2020/README.md:2]: welcome! could you please rename this file to another name that's not `README.md`? otherwise LGTM.

Comment (wikix) [reply shanghai/2020/README.md:2]: Got it, I will rename it.

**Pull request merged**